kpops/cli/main.py

```diff
@@ -191,7 +191,9 @@ def run_destroy_clean_reset(
     )
     for component in reversed(get_steps_to_apply(pipeline, steps)):
         log_action("Destroy", component)
-        component.destroy(dry_run=dry_run, clean=clean, delete_outputs=delete_outputs)
+        component.destroy(dry_run=dry_run)
```

Comment:
```suggestion
        component.destroy(dry_run)
```

less verbose calling

Reply:
or make `dry_run` a keyword-only argument for consistency. up to you